assets/js/base/context/cart-checkout/validation/index.js

```diff
@@ -151,12 +151,13 @@ export const ValidationContextProvider = ( { children } ) => {
 	 * Provides an id for the validation error that can be used to fill out
 	 * aria-describedby attribute values.
 	 *
-	 * @param {string} inputId The input css id the validation error is related
+	 * @param {string} errorId The input css id the validation error is related
 	 *                         to.
 	 * @return {string} The id to use for the validation error container.
 	 */
-	const getValidationErrorId = ( inputId ) => {
-		return inputId ? `validate-error-${ inputId }` : '';
+	const getValidationErrorId = ( errorId ) => {
+		const error = getValidationError( errorId );
+		return error && ! error.hidden ? `validate-error-${ errorId }` : '';
```

Comment:
this seems hard to understand and I'm not sure if it will return `validate-error-${ errorId }` or `''` or will return the expression `error && resultOfTernary` 
```suggestion
		return ( error && ! error.hidden ) ? `validate-error-${ errorId }` : '';
```

Reply:
Those parentheses are removed by ESLint. I refactored it to avoid using a ternary here, hope it makes code clearer: 5c2ff5597c2eda1c653052640973ebfd1d17cedc.